ohara-manager/build.gradle

```
@@ -54,6 +54,24 @@ task test {
   dependsOn setup
 }
 
+task it {
+  if(!project.hasProperty('skipManager')) {
+    doLast {
+      exec {
+        String port = 5050
```

for this case I prefer to throw exception (break the test at once) rather than use a default port. Since IT is impossible to work well with default port.

---

沒問題，我來修正一下

---

修正完成，請你再看一下，謝謝。